src/main/java/nextstep/subway/domain/section/SectionStationSorter.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.subway.domain.section;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SectionStationSorter {
+    public List<Long> getSortedStationIds(final List<Section> sections) {
+        List<Long> stationIds = new ArrayList<>();
+
+        Long currentStationId = getFisrtUpStation(sections).getUpStationId();
+        stationIds.add(currentStationId);
+        for (Section section : sections) {
+            if(!section.isFirst()){
+                stationIds.add(section.getUpStationId());
+            }
+            if(section.isLast()){
+                stationIds.add(section.getDownStationId());
+
+            }
+        }
+        return stationIds;
+    }
+
+    public Section getFisrtUpStation(final List<Section> sections) {
+        return sections.stream()
+                .filter(Section::isFirst)
+                .findAny()
+                .orElse(null);
```

Comment:
```suggestion
        return sections.stream()
                .filter(Section::isFirst)
                .findFirst()
                .orElseThrow(/** 오류 정의**/);
```
findAny보다는 findFirst가 더 어울리지 않을까하네요~ 그리고, 값이 없으면 예외 처리를 해주는게 더 코드를 단단하게 만든다고 봐요. 😊

Reply:
넵 NOT_FOUND_FIRST_EXCEPTION을 만들어서 예외처리를 하도록 수정하였습니다~